.github/workflows/contracts-compile-and-deploy-to-bridgenet.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           path: eth
 
       - name: Copy files to S3 AWS bucket
-        uses: Cardinal-Cryptography/github-actions/copy-file-to-s3@v1
+        uses: Cardinal-Cryptography/github-actions/copy-file-to-s3@v6
```

Comment:
Even though there's a default value to newly added `if-exists` argument in this action, this default value changes behavior (logc) if the action - now we skip when file exists, and previously we did Overwrite. Then I think we need to bump it to `v7`.

Reply:
I was mistaken, action does not change its behaviour.